src/bin/cargo/commands/bench.rs

```diff
@@ -16,6 +16,12 @@ pub fn cli() -> Command {
                 .num_args(0..)
                 .last(true),
         )
+        .arg_package_spec(
```

Comment:
In markdown help, the order is package, target, and feature selection. Should we make the consistent?